software/apps/audio_module/event_detector/Makefile

```diff
@@ -0,0 +1,17 @@
+# makefile for user application
+
+# the current directory
+APP_DIR := $(dir $(abspath $(lastword $(MAKEFILE_LIST))))
+
+# files needed for this code
+C_SRCS   := $(wildcard *.c) 
```

Comment:
Just a quick follow-up on this motivated by today's phone call. This should be largely fixed by the updates to the Tock build system. I updated signpost a few days ago in the `tock-master` branch, but we're holding off on merging that until after the paper deadline. I'll follow up again on April 11 :)